hooman/demos/cube3d.py

```diff
@@ -68,73 +66,47 @@ def yP(self, eye):
     Point(-50, 50, 50),
 ]
 
-
 dragging = False
 
-
 offset_x = hapi.WIDTH // 2
 offset_y = hapi.HEIGHT // 2
 
 while hapi.is_running:
-    bg_col = (255, 255, 255)
-    hapi.background(bg_col)
-    hapi.fill(0)
-
-    try:
-        hapi.stroke_size(3)
-        hapi.stroke(0)
-        for p1 in points:
-            for p2 in points:
-                hapi.line(
-                    offset_x + p1.x, offset_y + p1.y, offset_x + p2.x, offset_y + p2.y
-                )
-                # mouse_coords = (hapi.mouseX(), hapi.mouseY())
-                # eye = hapi.dist((0, 0), mouse_coords)
-                # if eye <= 0:
-                #     eye = 1
-                # hapi.line(50+p1.xP(eye), 50+ p1.yP(eye), 50+p2.xP(eye), 50+p2.yP(eye) )
-
-    except Exception as e:
-        raise
-    else:
-        pass
-    finally:
-        pass
-
-    # for i in range(8):
-    #     hapi.ellipse( 50+ points[i].x, 50+ points[i].y, 5, 5)
-
-    pmouseX = hapi.mouseX()
```

Comment:
you forgot to keep these 2 lines, very crucial as this variable is the previous mouse position